packages/block-library/src/group/deprecated.js

```diff
@@ -38,6 +39,79 @@ const migrateAttributes = ( attributes ) => {
 };
 
 const deprecated = [
+	// Version with no default content width.
```

Comment:
Why do we need the deprecation? The markup of the two different versions should be identical, so I wasn't sure if it needs a migration, since we don't want to make changes to existing blocks 🤔

Reply:
Again (and this might be connected to what I'm missing above?) if I don't add the migration, existing blocks with no layout set suddenly acquire a content width 😬

Reply:
Ah, excellent, thanks for confirming — at least knowing that much gives us a good base of behaviour for exploring our options 👍